README.md

```diff
@@ -1,10 +1,11 @@
 puppet-lint-absolute_classname-check
 ====================================
 
-[![Build Status](https://img.shields.io/travis/voxpupuli/puppet-lint-absolute_classname-check.svg)](https://travis-ci.org/voxpupuli/puppet-lint-absolute_classname-check)
-[![Gem Version](https://img.shields.io/gem/v/puppet-lint-absolute_classname-check.svg)](https://rubygems.org/gems/puppet-lint-absolute_classname-check)
-[![Gem Downloads](https://img.shields.io/gem/dt/puppet-lint-absolute_classname-check.svg)](https://rubygems.org/gems/puppet-lint-absolute_classname-check)
-[![Coverage Status](https://img.shields.io/coveralls/voxpupuli/puppet-lint-absolute_classname-check.svg)](https://coveralls.io/r/voxpupuli/puppet-lint-absolute_classname-check?branch=master)
+[![License](https://img.shields.io/github/license/voxpupuli/puppet-lint-spaceship_operator_without_tag-check.svg)](https://github.com/voxpupuli/puppet-lint-spaceship_operator_without_tag-check/blob/master/LICENSE)
+[![Test](https://github.com/voxpupuli/puppet-lint-spaceship_operator_without_tag-check/actions/workflows/test.yml/badge.svg)](https://github.com/voxpupuli/puppet-lint-spaceship_operator_without_tag-check/actions/workflows/test.yml)
+[![Release](https://github.com/voxpupuli/puppet-lint-spaceship_operator_without_tag-check/actions/workflows/release.yml/badge.svg)](https://github.com/voxpupuli/puppet-lint-spaceship_operator_without_tag-check/actions/workflows/release.yml)
+[![RubyGem Version](https://img.shields.io/gem/v/puppet-lint-spaceship_operator_without_tag-check.svg)](https://rubygems.org/gems/puppet-lint-spaceship_operator_without_tag-check)
+[![RubyGem Downloads](https://img.shields.io/gem/dt/puppet-lint-spaceship_operator_without_tag-check.svg)](https://rubygems.org/gems/puppet-lint-spaceship_operator_without_tag-check)
```

Comment:
puppet-lint-absolute_classname-check changed into puppet-lint-spaceship_operator_without_tag-check

Reply:
fixed it